client/src/rpc_response.rs

```diff
@@ -435,3 +435,9 @@ impl From<ConfirmedTransactionStatusWithSignature> for RpcConfirmedTransactionSt
         }
     }
 }
+
+#[derive(Serialize, Deserialize, Clone, Copy, Debug, PartialEq)]
+pub struct HighestSnapshotSlot {
+    pub full_snapshot_slot: Slot,
+    pub incremental_snapshot_slot: Option<Slot>,
+}
```

Comment:
mebbie something like this:
```suggestion
pub struct SnapshotSlotInfo {
    pub full: Slot,
    pub incremental: Option<Slot>,
}
```

Reply:
Can do.

Also, it looks like most of the RPC responses that are structs are prefixed with `Rpc`; should I do that here (i.e. `RpcSnapshotSlotInfo`)?

Reply:
Done!